api/r/cellxgene.census/tests/testthat/test-release_directory.R

```diff
@@ -1,6 +1,6 @@
 # A known-good Cell Census version. This may need updating if the version used
 # is withdrawn for any reason.
-KNOWN_CENSUS_VERSION <- "2023-02-13"
+KNOWN_CENSUS_VERSION <- "2023-04-04"
```

Comment:
Ah! This will be less of a problem once have a LTS (5 year) release.

Reply:
Why not just designate one build to be a "long term test build", give it a tag, and not delete it?   I don't think we need a "real" LTS release for this purpose....